keyboards/keychron/q0/q0.c

```diff
@@ -16,6 +16,20 @@
 
 #include "quantum.h"
 
+#ifdef ENCODER_ENABLE
+bool encoder_update_kb(uint8_t index, bool clockwise) {
+    if (!encoder_update_user(index, clockwise)) { return false; }
+    if (index == 0) {
+        if (clockwise) {
+            tap_code(KC_VOLU);
+        } else {
+            tap_code(KC_VOLD);
```

Comment:
Would prefer this form, since some systems have issues with media keys if they're sent too fast.  And this doesn't relay on having a tap code delay defined. 

```suggestion
            tap_code_delay(KC_VOLU, 10);
        } else {
            tap_code_delay(KC_VOLD, 10);
```

Reply:
As above.